cli/config/configfile/file_test.go

```diff
@@ -142,7 +143,43 @@ func TestConfigFile(t *testing.T) {
 	assert.Equal(t, configFilename, configFile.Filename)
 }
 
-func TestGetAllCredentials(t *testing.T) {
+type mockNativeStore struct {
+	GetAllCallCount int
+	EraseCalls      []string
+	GetCalls        []string
+	StoreCalls      []string
+
+	authConfigs map[string]types.AuthConfig
+}
+
+func (c *mockNativeStore) Erase(serverAddress string) error {
+	c.EraseCalls = append(c.EraseCalls, serverAddress)
+	delete(c.authConfigs, serverAddress)
+	return nil
+}
+
+func (c *mockNativeStore) Get(serverAddress string) (types.AuthConfig, error) {
+	return c.authConfigs[serverAddress], nil
+}
+
+func (c *mockNativeStore) GetAll() (map[string]types.AuthConfig, error) {
+	c.GetAllCallCount = c.GetAllCallCount + 1
```

Comment:
It looks like this field isn't being checked in the tests. I assume you want to check it is 0.